src/main/ruby/truffleruby/core/truffle/regexp_operations.rb

```diff
@@ -115,6 +116,9 @@ def self.match_in_region_tregex(re, str, from, to, at_start, encoding_conversion
         regex_result = compiled_regex.execBytes(str_bytes, from)
       end
       if bail_out
+        if USE_TRUFFLE_REGEX_WITH_WARN
+          warn "match_in_region_tregex(/#{re}/, \"#{str}\"@#{str.encoding}, #{from}, #{to}, #{at_start}, #{encoding_conversion}, #{start}) can’t be run as a Truffle regexp and fell back to Joni"
```

Comment:
A nice trick to avoid the manual quoting with `/` and `\"` is to use `.inspect` (also it will show Regexp options more clearly, `(?i-mx:ab)` vs `/ab/i`), I'll change to that.